src/app/timer/timer.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, OnInit } from '@angular/core';
+import { interval, subscribeOn, Subscription, timer, fromEvent, from } from 'rxjs';
+
+
+@Component({
+  selector: 'app-timer',
+  templateUrl: './timer.component.html',
+  styleUrls: ['./timer.component.css'],
+})
+export class TimerComponent implements OnInit {
+
+  //Ненужная переменная, просто для гита
+  bgit = 'Lorem'
+  
+  
+  constructor() {
+
+  }
+
+  hours = 0;
+  minuts = 0;
+  seconds = 0;
+
+  text:string = "pause"
+
+  date = new Date(this.hours, this.minuts, this.seconds);  
+
+  myTimer = timer(0, 1000);
+
+  time!: Subscription;
+
+  isWork = false;
```

Comment:
timerWorks, isTimeRunning is more understandable name (let's avoid some general words)